From a62322bcdd007200642e04335477f7d3a5d9a0dc Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Wed, 22 Sep 2021 19:41:13 +0200
Subject: [PATCH] Drop duplicate `ComputedEffectTiming` and `Animation` dfns

The dictionary and interface are partially re-defined in Level 2. Patch is only
needed because Level 2 is a delta spec. It will likely need to be kept around
for as long as that remains the case.
---
 ed/idl/web-animations.idl | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/ed/idl/web-animations.idl b/ed/idl/web-animations.idl
index 6e973d195..3f750de92 100644
--- a/ed/idl/web-animations.idl
+++ b/ed/idl/web-animations.idl
@@ -25,8 +25,6 @@ interface Animation : EventTarget {
              attribute DOMString                id;
              attribute AnimationEffect?         effect;
              attribute AnimationTimeline?       timeline;
-             attribute double?                  startTime;
-             attribute double?                  currentTime;
              attribute double                   playbackRate;
     readonly attribute AnimationPlayState       playState;
     readonly attribute AnimationReplaceState    replaceState;
@@ -86,10 +84,6 @@ enum FillMode { "none", "forwards", "backwards", "both", "auto" };
 enum PlaybackDirection { "normal", "reverse", "alternate", "alternate-reverse" };
 
 dictionary ComputedEffectTiming : EffectTiming {
-    unrestricted double  endTime;
-    unrestricted double  activeDuration;
-    double?              localTime;
-    double?              progress;
     unrestricted double? currentIteration;
 };
 
-- 
2.33.0.windows.2

